openmls/src/group/mls_group/proposal.rs

```diff
@@ -458,4 +453,123 @@ impl MlsGroup {
             .remove(proposal_ref)
             .ok_or(RemoveProposalError::ProposalNotFound)
     }
+
+    // === Create handshake messages ===
+    // TODO: share functionality between these.
```

Comment:
Maybe this should be an issue instead?

Reply:
I just copied this from `core_group` and didn't investigate further. But feel free to file an issue.

Reply:
Rather remove the comment completely, because I don't think it makes sense.